mailpoet/assets/css/src/components-automation-analytics/_table.scss

```diff
@@ -0,0 +1,29 @@
+.mailpoet-automation-analytics {
+  .woocommerce-table__header,
+  .woocommerce-table__item,
+  .woocommerce-table__empty-item {
+    text-wrap: balance;
+
+    @include respond-to(medium-screen) {
+      padding: 8px 12px;
+    }
+  }
+
+  .woocommerce-table__header.is-sortable {
+    padding: 2px;
+
+    button {
+      padding: 4px 20px;
+      width: 100%;
+
+      @include respond-to(medium-screen) {
+        padding: 4px 8px;
+      }
+    }
+  }
+
+  .woocommerce-table__header,
+  .woocommerce-table__header button {
+    font-weight: bold;
+  }
+}
```

Comment:
Not sure why this happens. But I think this view more element needs to be set absolute

```suggestion
}
.components-popover {
position: absolute;
}
```

Reply:
![image](https://github.com/mailpoet/mailpoet/assets/6458412/6c8e12e2-9288-448b-b44c-3a6dc9b030bd)
vs
![image](https://github.com/mailpoet/mailpoet/assets/6458412/5181cccd-b10d-4dde-963e-feb5e2278ab5)

Reply:
Ah! Had to rerun `pnpm install`, now its ok :ok_hand: sorry.